netlogo-core/src/main/lab/SpreadsheetExporter.scala

```diff
@@ -114,8 +123,13 @@ class SpreadsheetExporter(modelFileName: String,
     out.println()
     out.print(csv.header(if (protocol.runMetricsEveryStep || !protocol.runMetricsCondition.isEmpty) "[all run data]"
                               else "[initial & final values]"))
-    for(_ <- runs; metric <- protocol.metrics)
-      out.print(',' + csv.header(metric))
+    for(_ <- runs) {
+      if (shouldIncludeSteps)
+        out.print(',' + csv.header("tick"))
```

Comment:
In the table output when it lists the ticks it uses `[step]` as the header; I think we should match that here since the **Go commands** for the experiment might not advance the tick counter, or might advance it less than 1.